src/__tests__/prepare.test.js

```diff
@@ -44,6 +44,23 @@ end
 `);
 });
 
+describe('when the version.rb contains a prerelease version', () => {
+  it('writes the new version to the version.rb file', async () => {
+    await prepare(
+      {},
+      { ...context, nextRelease: { version: '1.0.0-alpha.1' } },
+      { versionFile, gemspec, gemName },
+    );
+    const versionContents = await readFile(path.resolve(cwd, versionFile), 'utf8');
+    expect(versionContents).toEqual(`# frozen_string_literal: true
+
+module TestGem
+  VERSION = '1.0.0-alpha.1'
```

Comment:
I think this won’t work with ruby gems, the pre versions must look like `1.0.0.alpha.1`, notice no dashes...